### PR TITLE
[#798] Rename Utils.removeItemToArray

### LINKED ISSUE #798: Fix odd method name ex.Util.removeItemToArray to ex.Util.removeItemFromArray
### Context
This is an oddly worded confusing method ex.Util.removeItem**To**Array, it just needs to be renamed. 

### Proposal
Rename it to ex.Util.removeItem**From**Array

### PR DESCRIPTION
**Pull request by Crizzooo**
Closes #798

## Changes:

- Utils.removeItemToArray -> Utils.removeItemFromArray
- Created test specs to test this function in UtilSpec.ts

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Added a new Interface `AbsolutePosition` which can described the `position` option. A `string` can also describe `position`
 ### Changed
 - Edge builds have more descriptive versions now containing build number and Git commit hash (e.g. `0.10.0-alpha.105#commit`) ([#777](https://github.com/excaliburjs/Excalibur/issues/777))
+- Changed Utils.removeItemToArray() to Utils.removeItemFromArray()
```
Comment (kamranayub) [inline CHANGELOG.md]: Please add a link to the issue, similar to the other Changelog entries 👍

**Comment by Crizzooo** [comment]
Updated the changeLog (I shouldve checked the other logs) okay now? Havent used TypeScript before - read some docs today and installed the apm linter for it, but did not run TSLint. Is there something else I need to do for this PR?

**Comment by kamranayub** [comment]
All the linting is done at build time, you can run the same build the CI runs by doing `npm run all`. You shouldn't need any extra tools, it's all included via npm. In Visual Studio Code, we also recommend installing the TSLint plugin so you can catch errors while you type. If you do want to run any whitespace fixes, you can commit them separately from code changes.

File: src/spec/UtilSpec.ts [hunk 2]
```diff
@@ -27,4 +27,22 @@ describe('Utility functions', () => {
...
+         var arrayToRemove = ['Stark', 'Lannister', 'Targaryen'];
+         expect(ex.Util.removeItemFromArray('Lannister', arrayToRemove)).toBe(true);
+         expect(arrayToRemove.length).toBe(2);
+      });
+
+      it('should return false when item to delete is not present', () => {
+         var arrayToRemove = ['Godfrey', 'Crizzo', 'Fullstack'];
+         expect(ex.Util.removeItemFromArray('Lannister', arrayToRemove)).toBe(false);
+      });
+   })
```
Comment (kamranayub) [inline src/spec/UtilSpec.ts]: Looks like just missing semi-colon here

Comment (Crizzooo) [inline src/spec/UtilSpec.ts]: Ah, I am so sorry I missed that. Changing now...

Also, when running those tests I get this error on lines 34, 39, 45 `src/spec/UtilSpec.ts(45,25): error TS2339: Property 'removeItemFromArray' does not exist on type 'typeof "Util/Index"'.` Not sure what is breaking... 

Thanks for bearing with me today as I make my first PR, get used to a massive code base, and learn TypeScript/better git workflow :). It's been a lot of fun and I'd like to make more meaningful contributions as I'm certainly learning a lot about contributing very quickly

**Comment by kamranayub** [comment]
No problem, we are happy to help and accept contributions!

**Pull request closed without merging**